Todo.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery'
+import { v4 as uuidv4 } from 'uuid'
+
+export class Todo {
+  constructor(text) {
+    this.isDone = false
+    this.text = text
+    this.id = uuidv4()
+  }
+
+  createNewTodo(text) {
+    const todo = $('<li>', {
+      class: 'todo__item',
+      text,
+      'data-id': this.id
+    })
+
+    this.id = todo.attr('data-id')
+
+    if (text === '') {
```

Comment:
so, you're creating new todo without the text and after that you're displaying alert with error. It should be outside of the class